### PR TITLE
Added openwisp admin theme to django-freeradius

### PR DESCRIPTION
**Pull request by anurag-ks**
Fix for - https://github.com/openwisp/django-freeradius/issues/96

### CODE REVIEW

**Comment by anurag-ks** [comment]
Yeah I did test it, anyway I'll do the required changes.

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ django-model-utils
 swapper
 djangorestframework>=3.7,<3.8
 passlib
+openwisp-utils
```
Comment (rohithasrk) [inline requirements.txt]: Also mentioning the versions is a good practice. Helps you handle compatibility.

File: tests/settings.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
     'django.contrib.staticfiles',
     'django.contrib.admin',
     'django_freeradius',
+    'openwisp_utils.admin_theme',
```
Comment (rohithasrk) [inline tests/settings.py]: Check the import order. Make sure you see the admin theme instead of default django admin page.

**Comment by anurag-ks** [comment]
@rohithasrk done the changes.

File: requirements-test.txt [hunk 3]
```diff
@@ -9,3 +9,4 @@ psycopg2
 sphinx
 sphinx_rtd_theme
 freezegun
+openwisp-utils==0.2
```
Comment (nemesisdesign) [inline requirements-test.txt]: use the form `openwisp-utils>=0.2,<0.3` as it's more maintainable

**Comment by rohithasrk** [comment]
You should be seeing something like this.

screenshot from 2018-02-19 01-30-16

**Comment by anurag-ks** [comment]
screenshot from 2018-02-19 11 56 09
screenshot from 2018-02-19 11 56 44
@rohithasrk I am getting this ^

**Comment by anurag-ks** [comment]
@rohithasrk nvm I got it - 
screenshot from 2018-02-19 12 01 14

**Pull request merged**